packages/react/src/class-names.js

```diff
@@ -30,6 +30,15 @@ let classnames = (args: Array<ClassNameArg>): string => {
         if (Array.isArray(arg)) {
           toAdd = classnames(arg)
         } else {
+          if (
+            process.env.NODE_ENV !== 'production' &&
+            arg.styles !== undefined
```

Comment:
Could we check `name` as well just in case someone has a class name called `styles`?

Reply:
added